### DISCUSSION TITLE
Integrations with release note products

### DISCUSSION THREAD

**Discussion by miniscruff** [miniscruff/changie#451] [Ideas]
I have been researching products that let you build release notes and changelogs using WYSIWYG style UIs and what features they support. Most of the features are UI/UX replacements for a CLI tool like text, markdown, html, emojis, images, etc. However, some ideas like UI widgets, email newsletters or SEO values would be much harder to build in a CLI. Since I have only ever used changie to build a markdown file to then dump into the static site portion of changie to be included in [this page](https://changie.dev/guide/changelog/) I do not have any experience with these tools.

I think it would be cool if changie was able to integrate with, or some how export to these tools but I have a feeling there would need to be some adjustments made since changie only ever outputs effectively a text file. 

This would only make sense to build if people were actually using it though, so if you have experience with any of these tools, or are considering using one of them in combination with Changie I would love to hear about it. I will also try and reach out to some of them to see if this combination would even make sense. ( Or on the crazy off chance any developer of one of these products sees this, hello! 👋 )

Example products in no particular order:
https://releasenotes.io/
https://announcekit.app/
https://kampsite.co/features/changelogs/
https://canny.io/features/product-changelog

**Comment by sheldonhull** [comment]
I use changing more for release notes, and true changelog entries because git provides the developer level log. 

For me, changie is more about communicating release notes. I've even thought about changing the default file to be `RELEASE.md`.
